src/Unosquare.Swan/Networking/JsonClient.cs

```diff
@@ -28,8 +28,9 @@ public class JsonClient
         /// <param name="payload">The payload.</param>
         /// <param name="authorization">The authorization.</param>
         /// <param name="ct">The cancellation token.</param>
-        /// <returns>The type of the result produced by this Task</returns>
-        /// <exception cref="System.Exception">Represents errors that occur during application execution</exception>
+        /// <returns>
+        /// The type of the result produced by this Task
```

Comment:
super generic...maybe something like "returns a task with a result of the requested type"